pixel-painting/bundle.js

```diff
@@ -0,0 +1,83 @@
+const client = new WebSocket("ws://" + window.location.host);
+
+const metadataLabel = 14;
+const query = (window.location.search || "?")
+  .substr(1)
+  .split('&')
+  .filter(x => x != '')
+  .map(x => x.split('='))
+const delay = Number((query.filter(([k, v]) => k == 'delay')[0] || [])[1] || 0);
```

Comment:
How does one remove himself from commit authors? Asking for a friend.

Reply:
🤣

Reply:
Trying to not degrade our street cred 

![Screenshot 2022-03-18 at 14 34 45](https://user-images.githubusercontent.com/79840582/159012514-e3595343-9ac2-4e31-b787-2af117ac9c93.png)